src/test/java/org/shemenev/v2/ReportGeneratorTest.java

```diff
@@ -0,0 +1,38 @@
+package org.shemenev.v2;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.net.NetworkInterface;
+import java.time.Duration;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static org.junit.Assert.*;
```

Comment:
Это плохая практика. Лучше явно испортировать те классы, которые ты используешь. Тебе нужен тольк assertTrue
```suggestion
import static org.junit.Assert.assertTrue;
```